code/modules/organs/organ_description.dm

```diff
@@ -196,3 +196,48 @@
 	min_broken_damage = 15
 	max_volume = 3
 	default_type = /obj/item/organ/external/unbreakable
+
+////SKELETON////
+/datum/organ_description/chest/skeletal
+	name = "ribcage"
+	max_damage = 20
+	min_broken_damage = 5
+	max_volume = 1
+	default_type = /obj/item/organ/external/skeletal/chest
+
+/datum/organ_description/groin/skeletal
+	name = "pelvis"
+	max_damage = 20
+	min_broken_damage = 5
+	max_volume = 1
+	default_type = /obj/item/organ/external/skeletal/groin
+
+/datum/organ_description/head/skeletal
+	max_damage = 20
+	min_broken_damage = 5
+	max_volume = 1
```

Comment:
Increase max volume so I can do a skeletal head and brain transplant and become sans undertale.